src/renderer/components/player/CaptionProgram.tsx

```diff
@@ -91,225 +90,207 @@ const COMMANDS : { [command : string] : (el: HTMLElement, value : string) => any
   },
 
-  blink: function(el : HTMLElement, value : string) {
-    return function(runNextCommand : Function) {
-      let fns : Function[] = [];
+  blink: function (el: HTMLElement, value: string, config: Config) {
+    return function (runNextCommand: Function) {
+      let fns: Function[] = [];
       let i = 0;
+      el.style.color = config.defaultScene.blinkColor;
+      el.style.fontSize = config.defaultScene.blinkFontSize + "vmin";
+      el.style.fontFamily = config.defaultScene.blinkFontFamily;
       el.className = "text-blink";
-      value.split('/').forEach(function(word) {
+      value.split('/').forEach(function (word) {
         word = word.trim();
         let j = i;
         i += 1;
-        fns.push(function() {
-          const showText = COMMANDS.showText(el, BLINK_DURATION + ' ' + word);
-          const wait = COMMANDS.wait(el, '' + BLINK_DELAY);
+        fns.push(function () {
+          const showText = COMMANDS.showText(el, BLINK_DURATION + ' ' + word, config);
+          const wait = COMMANDS.wait(el, '' + BLINK_DELAY, config);
           showText(function() { wait(fns[j + 1]); });
         })
       });
-      const lastWait = COMMANDS.wait(el, '' + BLINK_GROUP_DELAY);
-      fns.push(function() {
+      const lastWait = COMMANDS.wait(el, '' + BLINK_GROUP_DELAY, config);
+      fns.push(function () {
         lastWait(runNextCommand);
       });
       fns[0]();
     }
   },
 
-  cap: function(el : HTMLElement, value : string) {
-    const showText = COMMANDS.showText(el, CAPTION_DURATION + ' ' + value);
-    const wait = COMMANDS.wait(el, '' + CAPTION_DELAY);
-    return function(runNextCommand : Function) {
+  cap: function (el: HTMLElement, value: string, config: Config) {
+    const showText = COMMANDS.showText(el, CAPTION_DURATION + ' ' + value, config);
+    const wait = COMMANDS.wait(el, '' + CAPTION_DELAY, config);
+    return function (runNextCommand: Function) {
+      el.style.color = config.defaultScene.captionColor;
+      el.style.fontSize = config.defaultScene.captionFontSize + "vmin";
+      el.style.fontFamily = config.defaultScene.captionFontFamily;
       el.className = "text-caption";
       showText(function() { wait(runNextCommand); });
     }
   },
 
-  bigcap: function(el : HTMLElement, value : string) {
-    const showText = COMMANDS.showText(el, CAPTION_DURATION + ' ' + value);
-    const wait = COMMANDS.wait(el, '' + CAPTION_DELAY);
-    return function(runNextCommand : Function) {
+  bigcap: function (el: HTMLElement, value: string, config: Config) {
+    const showText = COMMANDS.showText(el, CAPTION_DURATION + ' ' + value, config);
+    const wait = COMMANDS.wait(el, '' + CAPTION_DELAY, config);
+    return function (runNextCommand: Function) {
+      el.style.color = config.defaultScene.captionBigColor;
+      el.style.fontSize = config.defaultScene.captionBigFontSize + "vmin";
+      el.style.fontFamily = config.defaultScene.captionBigFontFamily;
```

Comment:
Will these values always come from the "default scene" and not the current scene?

Reply:
Whoops, this is a hold-over from when this was only set through `Config` and not `Scene`
Fixed.